packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -397,6 +409,80 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
         },
         onDone: 'uploading',
       },
+      faceMovementCheck: {
```

Comment:
Would it be possible to keep the checks combined and instead just skip the `flashFreshnessColors` step with a conditional based on the name of the check?

For example the `checkMatch` step could conditionally choose to go to the freshness color steps based on the challenge name?

Alternatively we can add conditionals to the freshness color steps to just skip them if the challenge name doesnt match?

Reply:
Yes, updated to branch after `checkMatch` based on challenge type.